profiles/tests/test_validators.py

```diff
@@ -49,6 +49,34 @@ def execute_denied_number_test(number):
     def test_accept_valid_string(self, number):
         assert validate_finnish_national_identification_number(number) is None
 
+    @pytest.mark.parametrize(
+        "intermediary",
+        [
+            "+",
+            "-",
+            "A",
+            "B",
+            "C",
+            "D",
+            "E",
+            "F",
+            "U",
+            "V",
+            "W",
+            "X",
+            "X",
+        ],
+    )
+    def test_accept_valid_intermediary_letters(self, intermediary):
+        assert (
+            validate_finnish_national_identification_number(f"010101{intermediary}1231")
+            is None
+        )
+
+    @pytest.mark.parametrize("intermediary", ["_", "G", "1"])
+    def test_deny_invalid_intermediary_letters(self, intermediary):
+        self.execute_denied_number_test(f"010101{intermediary}1231")
```

Comment:
This test seems to be the same as the already existing `test_deny_invalid_century_character` test. They could be merged into one.

Reply:
Right-o, very much duplicated. Removed the other one.